src/components/personal-information/index.js

```diff
@@ -176,7 +188,7 @@ const PersonalInfoComponent = ({
                             }
                             <ReactTooltip id="promo-code-info">
                                 <div className={styles.moreInfoTooltip}>
-                                    Promo code will be applied to all tickets in this order.  If you wish to utilize more than one promo code, simply place another order after you complete this registration order. Only one promo code can be applied per order.
+                                    Your promo code will be applied to all tickets in this order. If you wish to use more than one promo code, simply place another order after you complete this order. Only one promo code can be applied per order.
```

Comment:
@edwinfinch please dont change verbiage, if u need to change it then u need to externalize the resource ( on another commit)
have in mind this will be back to main in order to be able to publish a version.

Reply:
What do you mean by "have in mind this will be back to main in order to be able to publish a version"?

Reply:
this commit will be merged back to main
so we cant change wording already approved by another clients
if u need to change some wording , u need allow to set it by an external prop and define as a default prop value the original text